modules/log6x-6-2-0-rn.adoc

```diff
@@ -6,52 +6,60 @@
 [id="logging-release-notes-6-2-0_{context}"]
 = Logging 6.2.0 Release Notes
 
-////
-TOFIX
-This release includes link:https://access.redhat.com/errata/RHBA-2024:9038[{logging-uc} {for} Bug Fix Release 6.2.0].
-////
+This release includes link:https://access.redhat.com/errata/RHBA-2025:2398[{logging-uc} {for} Bug Fix Release 6.2.0].
 
 [id="openshift-logging-release-notes-6-2-0-enhancements_{context}"]
 == New Features and Enhancements
 
-////
-TOFIX:
-=== Log Collection
-
-* This enhancement adds the source `iostream` to the attributes sent from collected container logs. The value is set to either `stdout` or `stderr` based on how the collector received it. (link:https://issues.redhat.com/browse/LOG-5292[LOG-5292])
 
-* With this update, the default memory limit for the collector increases from 1024 Mi to 2048 Mi. Users should adjust resource limits based on their cluster’s specific needs and specifications. (link:https://issues.redhat.com/browse/LOG-6072[LOG-6072])
+=== Log Collection
 
-* With this update, users can now set the syslog output delivery mode of the `ClusterLogForwarder` CR to either `AtLeastOnce` or `AtMostOnce.` (link:https://issues.redhat.com/browse/LOG-6355[LOG-6355])
+* With this update, HTTP outputs include a `proxy` field that you can use to send log data via an HTTP proxy. (link:https://issues.redhat.com/browse/LOG-6069[LOG-6069])
```

Comment:
Future reference, [according to the IBM Style Guide (ISG)](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v):
> Use (via), which means by way of, to refer to routing data, such as in “Messages are routed via SMTP”. In other situations, try to replace via with one of the following synonyms: across, along, by, from, on, or through.

Not worth holding up the merge, as fas as I am concerned.

Reply:
upated